system-int-test/test/cmr/system_int_test/search/collection_sorting_search_test.clj

```diff
@@ -379,3 +384,55 @@
          "sensor" [c1 c2 c3 c4 c5]
          ;; Descending sorts by the max value of a multi value fields
          "-sensor" [c2 c5 c1 c4 c3])))
+
+(deftest collection-fuzzy-end-date-test
+  (let [today (f/unparse (f/formatters :date-time) (t/now))
+        outside-ongoing (f/unparse (f/formatters :date-time)
```

Comment:
Might be useful to add one more date really far in the future like the year 4000 (beyond 3000) where the second sort will put it at the end to show that 3000 is used for its ongoing date.